site/src/sphinx/server-annotated-service.rst

```diff
@@ -180,6 +180,46 @@ one of the following supported types:
 - ``String``
 - ``Enum``
 
+Note that you can omit the value of `@Param`_ if you compiled your code with ``-parameters`` javac option.
+In this case the variable name is used as the value.
+
+.. code-block:: java
+
+    public class MyAnnotatedServiceClass {
+        @Get("/hello/{name}")
+        public HttpResponse hello1(@Param String name) { ... }
+    }
+
+.. note::
+
+    You can configure ``-parameters`` javac option with your build framework as follows.
```

Comment:
How about:

    You can configure your build tool to add ``-parameters`` javac option as follows.